Fabric.Identity.API/scripts/Install-Identity.ps1

```diff
@@ -34,6 +35,12 @@ if(!(Test-IsRunAsAdministrator))
 Write-DosMessage -Level "Information" -Message "Using install.config: $installConfigPath"
 $installSettingsScope = "identity"
 $installSettings = Get-InstallationSettings $installSettingsScope -installConfigPath $installConfigPath
+
+$idpssConfig = $installSettings.identityProviderSearchServiceConfig
+if($null -eq $idpssConfig) {
+    $idpssDirectoryPath = Get-WebConfigPath -service "IdentityProviderSearchService" -discoveryServiceUrl $installSettings.discoveryService -noDiscoveryService $noDiscoveryService -quiet $quiet
```

Comment:
Don't try to get path from installConfig, just get from Discovery service and IIS.